Code/GraphMol/Kekulize.cpp

```diff
@@ -347,6 +348,13 @@ bool kekulizeWorker(RWMol &mol, const INT_VECT &allAtms,
           // mol.debugMol(std::cerr);
           numBT++;
         } else {
+          // undo any remaining changes we made while here
```

Comment:
Just to be clear, this is triggered when we can't add double bonds and we also can't backtrack?